tests/functional.rs

```diff
@@ -517,6 +517,21 @@ async fn functional() {
         assert!(will_publish(test_msg, &ca4, &expected_files).await);
     }
 
+    // short hand to expect ROAs under CA4, re-added when parent comes back
+    // i.e. it now has RC 2 and 3, but no more 0 and 1
```

Comment:
This comment says "no more 0 and 1" but the caller passes in `roas` with value `&[route_rc0_1, route_rc1_1]` (i.e. referencing a 0 and a 1). Is this correct?

Reply:
oh right, well.. I think the rc0 and rc1 in the route definition are slightly misleading. They are under 10_0_0_0 and 10_1_0_0 respectively which happen to map to those RCs (until they become 2 and 3 that is). Changed the names to make this less confusing.